src/Components/Navbar.js

```diff
@@ -0,0 +1,33 @@
+import logoImg from '../img/logo.svg';
+import profileImg from '../img/profile.svg';
+import "../css/Navbar.css"
+
+function Profile({ user }) {
+    if (user.email) {
+        return ( 
+            <div className='navbar-profile'>
+                <img className='navbar-profile-img' alt='profile-img' src={profileImg}></img>
+                <div className="navbar-user">{user.email}</div>
+            </div>
+        )
+    } else {
+        return <a className="navbar-login" href="signin.html">로그인</a>;
+    }
+}
```

Comment:
@HMRyu 

한 파일에는 콤포넌트 하나만 구현해주세요!

또한 이 경우, 사용자 이메일 정보가 없으면 로그인 버튼이 렌더되도록 early return을 활용해보아도 좋겠습니다 

```suggestion
function Profile({ user }) {
   
    if (!user.email) {
            return <a className="navbar-login" href="signin.html">로그인</a>;
    } 
            return ( 
            <div className='navbar-profile'>
                <img className='navbar-profile-img' alt='profile-img' src={profileImg}></img>
                <div className="navbar-user">{user.email}</div>
            </div>
        )

}
```